apps/roam/src/utils/createReifiedBlock.ts

```diff
@@ -86,7 +95,7 @@ export const createReifiedBlock = async ({
 const RELATION_PAGE_TITLE = "roam/js/discourse-graph/relations";
 let relationPageUid: string | undefined = undefined;
 
-const getRelationPageUid = async (): Promise<string> => {
+export const getOrCreateRelationPageUid = async (): Promise<string> => {
```

Comment:
Why is this exported?  What's the use case as it relates to to this specific task?